### PR TITLE
54 header component 2

### PR DESCRIPTION
**Pull request by Tergel0820**

### CODE REVIEW

File: src/components/Cards.js [hunk 1]
```diff
@@ -54,43 +54,64 @@ export function Cards(props) {
...
-              <Button w="100" h="40" style={{ color: ' #4FADE0' }} onClick={onClick} type={'tertiary-outlined'}className="CardSmallText">Try Now</Button>
+            <div
+              style={{
+                display: "flex",
+                justifyContent: "flex-end",
+                width: "263px",
+              }}
+            >
+              {" "}
+              {/* <Button w="100" h="40" style={{ color: ' #4FADE0' }} onClick={onClick} type={'tertiary-outlined'}className="CardSmallText">Try Now</Button> */}
```
Comment (namuun1214) [inline src/components/Cards.js:65]: eniig yagaad comment bolgotsiin?

File: src/components/Cards.js [hunk 2]
```diff
@@ -54,43 +54,64 @@ export function Cards(props) {
...
+              {smallDescription}
+            </div>
           </div>
         </Padding>
 
         <Padding size={[0, 0, 20, 0]}>
-          {' '}
-          <Button onClick={onClick} style={{ color: ' #4FADE0' }} className="CardSmallText"w="100" h="40" type={'tertiary-outlined'}>Try Now</Button>
+          {" "}
+          {/* <Button onClick={onClick} style={{ color: ' #4FADE0' }} className="CardSmallText"w="100" h="40" type={'tertiary-outlined'}>Try Now</Button> */}
```
Comment (namuun1214) [inline src/components/Cards.js:87]: ene bas?

File: src/components/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            <div>Мэдээ</div>
+            <div>хандив өгөх</div>
+          </HeaderLinks>
+        </HeaderMenuCon>
+      </HeaderItems>
+    </HeaderStyle>
+  );
+};
+
+export { Header };
```
Comment (guyeg1) [inline src/components/Header.jsx:41]: export default Header;

File: src/assets/icons/vectorIcon.js [hunk 4]
```diff
@@ -2,7 +2,7 @@ import React from "react"
 export const VectorIcon = ( )=>{
     return (
         <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
-            <path fill-rule="evenodd" clip-rule="evenodd" d="M7.29289 0.292893C7.68342 -0.0976311 8.31658 -0.0976311 8.70711 0.292893L15.7071 7.29289C16.0976 7.68342 16.0976 8.31658 15.7071 8.70711L8.70711 15.7071C8.31658 16.0976 7.68342 16.0976 7.29289 15.7071C6.90237 15.3166 6.90237 14.6834 7.29289 14.2929L12.5858 9H1C0.447715 9 0 8.55228 0 8C0 7.44772 0.447715 7 1 7H12.5858L7.29289 1.70711C6.90237 1.31658 6.90237 0.683417 7.29289 0.292893Z" fill="black"/>
+            <path fillRule="evenodd" clipRule="evenodd" d="M7.29289 0.292893C7.68342 -0.0976311 8.31658 -0.0976311 8.70711 0.292893L15.7071 7.29289C16.0976 7.68342 16.0976 8.31658 15.7071 8.70711L8.70711 15.7071C8.31658 16.0976 7.68342 16.0976 7.29289 15.7071C6.90237 15.3166 6.90237 14.6834 7.29289 14.2929L12.5858 9H1C0.447715 9 0 8.55228 0 8C0 7.44772 0.447715 7 1 7H12.5858L7.29289 1.70711C6.90237 1.31658 6.90237 0.683417 7.29289 0.292893Z" fill="black"/>
         </svg>
     )
 }
```
Comment (guyeg1) [inline src/assets/icons/vectorIcon.js:8]: code-oo pull hiij awaaagu huuchnaar ni ywuulsn bn = assets dotor icons orno, icon.js -uud dutuu

**Pull request merged**